src/main/java/biz/paluch/logging/gelf/intern/GelfMessage.java

```diff
@@ -29,6 +29,8 @@
  */
 public class GelfMessage {
 
+    private static final Random rand = new Random();
```

Comment:
Let's use `ThreadLocalRandom` in `getRandomLong`.

Reply:
I see your point; I wasn't aware of the potential performance issues with Random in multithreaded applications. However, ThreadLocalRandom was introduced in Java 1.7 and you have maven.compiler.target and maven.compiler.source set to 1.6.
Maybe version 1.14.0 should end Java 6 support. Just for comparison, Log4j2 requires Java 7 since version 2.4 which was released in 2015.
Do you want me to change to 1.7 or do you have an alternative to ThreadLocalRandom?

Reply:
Given that newer Java versions start removing backwards-compilation capabilities, it makes sense to switch to Java 1.7.

Moving this ticket to logstash-gelf 1.14.0.